### DISCUSSION TITLE
FYI FabGL now has composite too!

### DISCUSSION THREAD

**Discussion by ghost** [Roger-random/ESP_8_BIT_composite#34] [General]
Good times for people interested in composite output....

https://www.youtube.com/watch?v=5QteUItkyVs

https://github.com/fdivitto/FabGL/discussions/192

**Comment by Roger-random** [comment]
I looked over FabGL briefly before I started this project. (https://newscrewdriver.com/2021/05/06/window-shopping-fabgl/)

FabGL wasn't set up for other people to implement rendering on different hardware, so I moved on to Adafruit GFX library. But now the author has tackled composite video directly. This is fantastic!

I loaded up FabGL composite video example for a quick look. (https://github.com/fdivitto/FabGL/blob/master/examples/Composite/BaseExample/BaseExample.ino)
NOTE: this does not compile on the latest official release of FabGL v1.0.6, I had to clone the repository directly to my Arduino library folder.

The color is not very steady on my little test TV but I don't know how much of that is my hardware vs. issues that could be solved by FabGL improvements yet to come.

Great stuff.
